libcst/metadata/tests/test_scope_provider.py

```diff
@@ -111,6 +111,25 @@ def fn_def():
         wrapper = MetadataWrapper(cst.parse_module("def a():\n    from b import c\n\n"))
         wrapper.visit(DependentVisitor())
 
+    def test_fstring_accesses(self) -> None:
+        m, scopes = get_scope_metadata_provider(
+            """
+            from a import b
+            f"{b}" "hello"
+            """
+        )
+        global_scope = scopes[m]
+        self.assertIsInstance(global_scope, GlobalScope)
+        global_accesses = list(global_scope.accesses)
+        self.assertEqual(len(global_accesses), 1)
+        import_node = ensure_type(
+            ensure_type(m.body[0], cst.SimpleStatementLine).body[0], cst.ImportFrom
+        )
+        b_referent = list(global_accesses[0].referents)[0]
+        self.assertIsInstance(b_referent, Assignment)
+        assert isinstance(b_referent, Assignment)  # for the typechecker's eyes
+        self.assertEqual(b_referent.node, import_node)
```

Comment:
nit: can we do this instead? We usually don't use assert because it's a noop in optimized mode.

```
if isinstance(b_referent, Assignment):
    self.assertEqual(b_referent.node, import_node)
```

Reply:
sure, but it doesn't matter much because there's a
```
self.assertIsInstance(b_referent, Assignment)
```
right above it, so if the assertion fails, no further code will be executed